flink-staging/flink-gelly/src/main/java/org/apache/flink/graph/Graph.java

```diff
@@ -282,6 +282,58 @@ public void flatMap(Edge<K, EV> edge, Collector<Tuple1<K>> out) {
 	}
 
 	/**
+	* Creates a graph from CSV files.
+	*
+	* Vertices with value are created from a CSV file with 2 fields
+	* Edges with value are created from a CSV file with 3 fields
+	* from Tuple3.
+	*
+	* @param path1 path to a CSV file with the Vertices data.
+	* @param path2 path to a CSV file with the Edges data
+	* @param context the flink execution environment.
+	* @return An instance of {@link org.apache.flink.graph.GraphCsvReader} , which on calling types() method to specify types of the
+	*		 Vertex ID, Vertex Value and Edge value returns a Graph
+	*/
+
+	public static  GraphCsvReader fromCsvReader(String path1, String path2, ExecutionEnvironment context){
```

Comment:
I would rename `path1` and `path2` to something like `verticesPath` and `edgesPath`